tensorboard/plugins/graph/graphs_plugin_v2_test.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Integration tests for the Graphs Plugin for TensorFlow v2."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import os.path
+
+from google.protobuf import text_format
+import tensorflow as tf
+
+from tensorboard.compat.proto import graph_pb2
+from tensorboard.plugins.graph import graphs_plugin_test
+
+
+tf.enable_v2_behavior()
+
+
+class GraphsPluginV2Test(graphs_plugin_test.GraphsPluginBaseTest, tf.test.TestCase):
+
+  def generate_run(self, run_name, include_graph, include_run_metadata):
+    x, y = np.ones((10, 10)), np.ones((10, 1))
+    val_x, val_y = np.ones((4, 10)), np.ones((4, 1))
+
+    model = tf.keras.Sequential([
+        tf.keras.layers.Dense(10, activation='relu'),
+        tf.keras.layers.Dense(1, activation='sigmoid')])
+    model.compile('rmsprop', 'binary_crossentropy')
+
+    model.fit(
+        x,
+        y,
+        validation_data=(val_x, val_y),
+        batch_size=2,
+        epochs=1,
+        callbacks=[tf.keras.callbacks.TensorBoard(
+            log_dir=os.path.join(self.logdir, run_name),
+            write_graph=include_graph)])
+
+  def _get_graph(self, *args, **kwargs):
+    """Fetch and return the graph as a proto."""
+    (graph_pbtxt, mime_type) = self.plugin.graph_impl(*args, **kwargs)
+    self.assertEqual(mime_type, 'text/x-protobuf')
+    return text_format.Parse(graph_pbtxt, graph_pb2.GraphDef())
+
+  def test_info(self):
+    raise self.skipTest('tf-nightly==20190222 does not write a conceptual graph.')
```

Comment:
Can you add a TODO to enable this?  Maybe e.g. as

`raise self.skipTest('TODO: enable this after tf-nightly writes a conceptual graph')`